components/specificity-meter/ui/src/main/resources/PhenoTips/SpecificityMetricWidget.xml

```diff
@@ -237,9 +239,10 @@
       var value = this._crtValue * 5;
       this.meterDisplay.title = this.levels[Math.round(value)] + " (" + value.toPrecision(3) + "/5)";
       this.meterDisplay.update(new Element('div', {'id' : "specificity-meter-value", "style" : "width: " + Math.round(v * 100) + "%;"}));
-      if (oldValue !== undefined &amp;&amp; oldValue != this._crtValue) {
+      if (oldValue !== undefined &amp;&amp; (!this._firedUpdate || oldValue != this._crtValue)) {
```

Comment:
It would not, because in case the actual score is zero, the comparison would 0 !== 0.